workspace.json

```diff
@@ -1915,6 +1915,29 @@
         }
       }
     },
+    "application-templates-login-service": {
+      "root": "libs/application/templates/login-service",
+      "sourceRoot": "libs/application/templates/login-service/src",
+      "projectType": "library",
+      "architect": {
+        "lint": {
+          "builder": "@nrwl/linter:eslint",
+          "options": {
+            "lintFilePatterns": [
+              "libs/application/templates/login-service/**/*.{ts,tsx,js,jsx}"
+            ]
+          }
+        },
+        "test": {
+          "builder": "@nrwl/jest:jest",
+          "outputs": ["coverage/libs/application/templates/login-service"],
+          "options": {
+            "jestConfig": "libs/application/templates/login-service/jest.config.js",
+            "passWithNoTests": true
+          }
+        }
+      }
+    },
```

Comment:
You might want to add the `extract-strings` (https://docs.devland.is/libs/localization#message-extraction) script here to handle messages to Contentful